### PR TITLE
Add SnapshotCollector and missing loggers

### PR DESCRIPTION
**Pull request by pharring**
Taking "sc" for Snapshot Collector.
Also adding some missing values from other logging projects and added links to their nuget packages because the "releases" link just gives links to source code.

### CODE REVIEW

File: EndpointSpecs/SDK-VERSIONS.md [hunk 1]
```diff
@@ -146,4 +173,10 @@ ASP.NET core SDK
 
 Public versions: https://github.com/Microsoft/ApplicationInsights-aspnetcore/releases
 
+### sc
```
Comment (clguimanMSFT) [inline EndpointSpecs/SDK-VERSIONS.md]: Is it useful to have a custom prefix when snapshot collector is auto-enabled by the App Service AppInsights site extension?

Comment (pharring) [reply EndpointSpecs/SDK-VERSIONS.md]: Perhaps. How about `scx` for that? I presume the implementation would test for the presence of the appropriate environment variable.

Comment (SergeyKanzhelev) [reply EndpointSpecs/SDK-VERSIONS.md]: Consider using prefixes for this. For Azure Web Apps prefix may be set automatically. internal link: https://mseng.visualstudio.com/AppInsights/Chuck/_git/StatusMonitor?path=%2FExtensions%2FExtension%2FAzureAppService%2FAzureAppService%2FSdkSourceTelemetryInitializer.cs&version=GBmaster&_a=contents&line=22&lineStyle=plain&lineEnd=22&lineStartColumn=8&lineEndColumn=53

Comment (pharring) [reply EndpointSpecs/SDK-VERSIONS.md]: Thanks. We'll use a prefix. No need to document it here.

**Comment by pharring** [comment]
Ready to merge (I don't have write access).

**Pull request merged**